src/stylesheets/components/_section.scss

```diff
@@ -1,37 +1,37 @@
 .usa-section {
-  @include u-padding-y($site-margins);
+  @include u-padding-y(units(4));
 
-  @include media($medium-screen) {
-    @include u-padding-y(6rem);
+  @include at-media('tablet') {
+    @include u-padding-y(units(8));
   }
 }
 
 .usa-section-light {
-  background-color: $color-gray-lightest;
+  background-color: color('base-lightest');
 }
 
 .usa-section-dark {
-  background-color: $color-primary-darkest;
-  color: $color-white;
+  background-color: color('primary-darker');
+  color: color('white');
```

Comment:
Can we can just use `black` or `white` for those two colors instead of a function?

Reply:
We could, but I'm not sold that we should, as this ties it back to the variable we defined as white and black.

Reply:
I don't feel strongly about this, but I _generally_ feel that the system shouldn't set a definition of white of black that isn't `white` or `black` — that is to say, I'm comfortable that we have white and black defined in the system but i don't think they should ever change from pure white and pure black, so maybe they're redundant but also who knows it's been a day 🤷‍♀️

Reply:
so let's just leave them as a function, at least for consistency's sake